### ISSUE TITLE
Lint proper use of jsx pragma

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno_lint#969]
```js
/** @jsxImportSource https://esm.sh/preact */
/** @jsx h */
/** @jsxFragment Fragment */
```

The above code works fine (it uses the `@jsxImportSource`) as expected. It is not obvious to users what should happen here though, so we should add a `deno lint` to disallow this. One or the other, not both.

_Originally posted by @lucacasonato in https://github.com/denoland/deno/issues/12631#issuecomment-962707286_
